### DISCUSSION TITLE
Webpack 5 Just Incompatible?

### DISCUSSION THREAD

**Discussion by programmersRuleTheWorld** [Glench/ExtPay#184] [Q&A] [answered]
Hey everybody, I'm Robbie.
I've been trying to get this to work for a few days, but it seems like no matter how I attempt to do it, it just won't work.

![image](https://github.com/Glench/ExtPay/assets/48508281/e09c1ad9-33d1-4d9c-b86b-52c961a332d5)

I have also tried without a bundler, just using serviceWorker as a module, and this is the result:

![image](https://github.com/Glench/ExtPay/assets/48508281/6bfe6745-8fa3-4d61-89ed-7745e9af1989)

Very frustrating. This seems like such a cool tool too.

**Comment by bilalsattar24** [comment, marked as answer]
Hey, are you still having this issue? I was able to get this to work with webpack. I just had to use it in my background.js file as 

```
import ExtPay from 'extpay';

var extpay = ExtPay(extensionPayAppName);

extpay.startBackground();
```
